falcon/request.py

```diff
@@ -1773,14 +1773,15 @@ class RequestOptions(object):
                 encoded according to the standard W3C algorithm (see
                 also http://goo.gl/6rlcux).
 
-        auto_parse_qs_csv: Set to ``False`` to treat commas in a query
-            string value as literal characters, rather than as a comma-
-            separated list (default ``True``). When this option is
-            enabled, the value will be split on any non-percent-encoded
-            commas. Disable this option when encoding lists as multiple
-            occurrences of the same parameter, and when values may be
-            encoded in alternative formats in which the comma character
-            is significant.
+        auto_parse_qs_csv: Pass in ``True`` to split
+            a query string value on any non-percent-encoded commas,
+            this option is disabled by default (``False``).
+            When ``False`` query strings containing commas are untouched,
+            his enables encoding lists as multiple of the same parameter
+            and cases where the comma character is significant
+            such as alternate encodings (i.e. ``/?t=1,2,3&t=4 becomes ['1,2,3', '4']``).
+            When ``True``, the query string value is split on the commas
+            and returned as a list (i.e. ``/?t=1,2,3&t=4 becomes ['1', '2', '3', '4']``).
```

Comment:
Same as above: *becomes* inbetween should not lie within backticks, see the examples by @nZac

Reply:
@vytas7 Sorry, my bed. It's already fixed.